doc/basics.md

```diff
@@ -933,60 +933,38 @@ format. If your JSON documents all contain arrays or objects, we even support di
 concatenation without whitespace. The concatenated file has no size restrictions (including larger
 than 4GB), though each individual document must be no larger than 4 GB.
 
-Here is a simple example, given `x.json` with this content:
-
-```json
-{ "foo": 1 }
-{ "foo": 2 }
-{ "foo": 3 }
-```
-
-```c++
-dom::parser parser;
-dom::document_stream docs = parser.load_many("x.json");
-for (dom::element doc : docs) {
-  cout << doc["foo"] << endl;
-}
-// Prints 1 2 3
-```
-
-In-memory ndjson strings can be parsed as well, with `parser.parse_many(string)`:
-
+Here is a simple example:
 
 ```c++
-dom::parser parser;
-  auto json = R"({ "foo": 1 }
-{ "foo": 2 }
-{ "foo": 3 })"_padded;
-dom::document_stream docs = parser.parse_many(json);
-for (dom::element doc : docs) {
-  cout << doc["foo"] << endl;
+auto json = R"({ "foo": 1 } { "foo": 2 } { "foo": 3 } )"_padded;
```

Comment:
If it is not already the case, please add a copy of this code example to our tests. Too often we have offered code examples in our documentation that contain minor typos and these are very annoying to newcomers.

Reply:
(To be clear, I do not mean that you have offered broken examples... I mean people like me.)

Reply:
Added it!